pages/home/home.ejs

```diff
@@ -77,6 +77,12 @@
                         <a href="https://prairielearn.readthedocs.io/en/latest" class="btn btn-<%= isAuthenticated ? 'xs my-0 align-text-bottom' : 'sm mb-1' %> btn-outline-primary justify-content-center">
                           View docs
                         </a>
+                        <a href="<%= plainUrlPrefix %>/scrap_paper" class="btn btn-<%= isAuthenticated ? 'xs my-0 align-text-bottom' : 'sm mb-1' %> btn-outline-primary justify-content-center">
```

Comment:
I would recommend disabling this until we can get feedback from a small group of testing users.